content/en/docs/components/katib/user-guides/metrics-collector.md

```diff
@@ -80,3 +84,64 @@ To define the metrics collector for your Experiment:
    recall=0.55
    precision=.5
    ```
+
+## Push-based Metrics Collector
+
+Your training code needs to call [`report_metrics`](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/report_metrics.py#L26) function in Python SDK to record metrics.
+
+To define the push-based metrics collector for your Experiment, you have two options:
+
+- YAML File
+
+    1. Specify the collector type `Push` in the `.collector.kind` field.
+
+    2. Write code in your training container to call `report_metrics` to report metrics.
+
+- [`tune`](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L166) function
+
+    Use tune function and specify the `metrics_collector_config` field. You can reference to the following example:
```

Comment:
You might need to explain how `report_metrics()` functions works separately. E.g. user needs to install `kubeflow-katib` SDK in their environment, and the `report_metrics()` will automatically add the timestamp for metrics.

Reply:
SGTM. I'll add the explanation to `report_metrics()`.